posthog/api/error_tracking.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from rest_framework import serializers, viewsets
+
+from posthog.api.forbid_destroy_model import ForbidDestroyModel
+from posthog.api.routing import TeamAndOrgViewSetMixin
+from posthog.models.error_tracking import ErrorTrackingGroup
+
+
+class ErrorTrackingGroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ErrorTrackingGroup
+        fields = [
+            "assignee",
+        ]
+
+    def update(self, instance: ErrorTrackingGroup, validated_data: dict, **kwargs) -> ErrorTrackingGroup:
```

Comment:
as mr activity log this makes me think... why no activity log :)

Reply:
😅 
I did skip over it for the first version. Will follow up tomorrow in a separate PR. I made the mistake of putting too many changes in one PR last week